collectors/apps.plugin/apps_groups.conf

```diff
@@ -314,12 +316,29 @@ airflow: *airflow*
 # -----------------------------------------------------------------------------
 # GUI
 
-gui: X Xorg xinit lightdm xdm gkrellm xfwm4 xfdesktop xfce* Thunar
-gui: xfsettingsd xfconfd gnome-* gdm gconf* dconf* xfconf* *gvfs gvfs* slim
-gui: *kdeinit* kdm plasmashell
-gui: evolution-* firefox chromium opera vivaldi-bin epiphany WebKit*
-gui: '*systemd --user*' chrome *chrome-sandbox* *google-chrome* *chromium* *firefox*
-gui: colord seatd greetd wayfire sway weston cage
+X: X Xorg xinit xdm Xwayland xsettingsd
+wayland: swaylock swayidle waypipe wayvnc
+kde: *kdeinit* kdm sddm plasmashell startplasma-* kwin* kwallet* krunner kactivitymanager*
+gnome: gnome-* gdm gconf* mutter
+mate: mate-* msd-* marco*
+cinnamon: cinnamon* muffin
+xfce: xfwm4 xfdesktop xfce* Thunar xfsettingsd xfconf*
+lxde: lxde* startlxde lxdm lxappearance* lxlauncher* lxpanel* lxsession* lxsettings*
+lxqt: lxqt* startlxqt
+enlightenment: entrance enlightenment*
+i3: i3*
+awesome: awesome awesome-client
+dwm: dwm.*
+sway: sway
+weston: weston
+cage: cage
+wayfire: wayfire
+gui: lightdm colord seatd greetd gkrellm slim qingy dconf* *gvfs gvfs*
+gui: '*systemd --user*' xdg-* at-spi-*
+
+webbrowser: *chrome-sandbox* *google-chrome* *chromium* *firefox* vivaldi* opera* epiphany
+webbrowser: lynx elinks w3m w3mmee
```

Comment:
Please, add `links` to this list.

Reply:
No idea how I missed that, it’s not like it’s the oldest text mode browser still under active development...